benchmarks/AclBench.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace LaminasBench\PermissionsAcl;
+
+use Laminas\Permissions\Acl\Acl;
+use Laminas\Permissions\Acl\Role\GenericRole;
+
+/**
+ * @Revs(2)
+ * @Iterations(2)
+ * @Warmup(1)
+ */
+class AclBench
+{
+    private const NUM_ROLES_WITHOUT_PARENT = 135;
+    private const NUM_ROLES_WITH_PARENT = 70;
+    private const NUM_RESOURCES = 324;
+    private const NUM_ALLOW_CALLED = 300;
+    private const NUM_DENY_CALLED = 300;
+
+    private Acl $acl;
+
+    public function __construct()
+    {
+        $this->acl = new Acl();
+    }
+
+    public function benchSetService(): void
```

Comment:
Overall OK test scenario, naming could be improved perhaps.

This is good as-is, but in future (also for your own benchmarks), remember to split scenarios and use-cases like you would do with a unit test.

This scenario does an "all at once", heh 😁